gymnasium/spaces/discrete.py

```diff
@@ -55,7 +56,7 @@ def is_np_flattenable(self):
         """Checks whether this space can be flattened to a :class:`spaces.Box`."""
         return True
 
-    def sample(self, mask: MaskNDArray | None = None) -> int:
+    def sample(self, mask: MaskNDArray | None = None) -> np.int64 | NDArray[np.int64]:
```

Comment:
I still think this should just be `np.int64`

Reply:
https://github.com/Farama-Foundation/Gymnasium/pull/327#discussion_r1104678853